src/MagnumPlugins/AssimpImporter/AssimpImporter.cpp

```diff
@@ -448,6 +471,18 @@ void AssimpImporter::doOpenData(const Containers::ArrayView<const char> data) {
             }
         }
     }
+
+    /* Before https://github.com/assimp/assimp/commit/e3083c21f0a7beae6c37a2265b7919a02cbf83c4
+       Assimp incorrectly read spline tangents as values in glTF animation tracks.
+       Quick and dirty check to see if we're dealing with a possibly affected file
+       and Assimp version. This might produce false-positives on files without
+       spline-interpolated animations, but for doOpenState and doOpenFile we 
+       have no access to the file content to check if the file contains "CUBICSPLINE". */
+    if (_f->scene->HasAnimations() && _f->importerIsGltf && ASSIMP_VERSION < 20201123) {
+        Warning{} << "Trade::AssimpImporter::open(): spline-interpolated animations imported" <<
+            "from this file are most likely broken using this version of Assimp. Consult the" <<
+            "importer documentation for more information.";
```

Comment:
```suggestion
        Warning{} << "Trade::AssimpImporter::openData(): spline-interpolated animations imported "
            "from this file are most likely broken using this version of Assimp. Consult the "
            "importer documentation for more information.";
```

(Using a single string literal instead.)

Reply:
Done